### PR TITLE
UI - Logic to manually drop items

### PR DESCRIPTION
**Pull request by niicojs**
### Short Description:

UI - Logic to manually drop items
### Changes:
- Add socket event to manually drop items from inventory

@OpenPoGo/maintainers

### CODE REVIEW

**Comment by meyer9** [comment]
:+1:

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/299/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

File: plugins/socket/uievents.py [hunk 1]
```diff
@@ -122,15 +121,15 @@ def client_ask_for_transfer(evt):
...
+                log("Transferring {0} (#{1}) with CP {2} and IV {3}".format(pokemon_name,
+                                                                            pokemon_num,
+                                                                            pokemon_cp,
+                                                                            pokemon_potential), color="green")
 
     @socketio.on("evolve_pokemon", namespace="/event")
     def client_ask_for_evolve(evt):
         if "bot" in state:
-            logger.log("Web UI action: Evolve", "yellow", fire_event=False)
+            logger.log("Web UI action: Evolve", color="yellow")
```
Comment (wchill) [inline plugins/socket/uievents.py]: Inconsistent use of `log` vs `logger.log`

File: plugins/socket/uievents.py [hunk 2]
```diff
@@ -12,19 +12,22 @@ def find(f, seq):
             return item
     return None
 
+def log(text, color='yellow'):
+    logger.log(text, color=color, prefix='UI')
```
Comment (wchill) [inline plugins/socket/uievents.py]: Socket/web stuff should use `fire_event=False`.

**Comment by wchill** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/299/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**